package.json

```diff
@@ -24,6 +28,7 @@
     "bootstrap-daterangepicker": "^3.0.3",
     "bootstrap-select": "^1.13.12",
     "cable_ready": "^4.0.7",
+    "chart.js": "^2.9.3",
```

Comment:
What's the difference between what this offers and chartist?

Reply:
Up to you